src/composables/Project.js

```diff
@@ -589,3 +589,38 @@ export async function gitCommit(projectId, message) {
     message,
   });
 }
+
+export async function gitGlobalSave(project) {
+  const nowDate = new Date();
+  const currentBranch = await getCurrentBranch(project.id);
+  const newBranch = `leto-modelizer_${nowDate.getTime()}`;
+
+  await createBranchFrom(
+    project.id,
+    newBranch,
+    currentBranch,
+    true,
+  );
+
+  const files = (await getStatus(project.id))
+    .filter((file) => file.hasUnstagedChanged
+      || file.isUntracked
+      || file.isUnstaged
+      || file.isStaged)
+    .map((file) => file.path);
+
+  await gitAdd(project.id, files);
+
+  await gitCommit(
+    project.id,
+    `leto-modelizer ${nowDate.toDateString()}`,
+  );
+
+  await gitPush(
+    project,
+    newBranch,
+    true,
+  );
+
+  FileEvent.GlobalSaveFilesEvent.next();
```

Comment:
to be compliant with other function please make a return here

Reply:
useless but done